docs-site/docs/development/useful_commands.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 3
+id: useful_commands
+title: Useful Commands for Development
+---
+
+### Drop, rebuild and seed db
+
+``` bash
+bin/rails db:drop db:create db:migrate db:seed
+```
+
+### Create new credentials
+
+``` bash
+rm config/credentials.yml.enc
+EDITOR="vim --nofork" bin/rails credentials:edit
+```
+
+### Run this additional process to have UI changes updated as they are changed
```

Comment:
**nitpick (blocking):** Might want to add here that you only need to run this command if you start the server with `bin/rails s` and do not use `bin/dev`

Reply:
Added a line to provide more context here.